packages/examples/sdk-server/src/index.ts

```diff
@@ -14,24 +15,26 @@ export const sdkServer = async (): Promise<FastifyInstance> => {
   const server = fastify();
 
   // Initialize SDK
-
   let configJson: Record<string, any> = {};
 
   try {
     configJson = JSON.parse(process.env.NXTP_CONFIG || "");
   } catch (e: unknown) {
     console.info("No NXTP_CONFIG exists, using config file and individual env vars");
   }
-  try {
-    let json: string;
-    const path = process.env.NXTP_CONFIG_FILE ?? "config.json";
-    if (fs.existsSync(path)) {
-      json = fs.readFileSync(path, { encoding: "utf-8" });
-      configJson = JSON.parse(json);
+
+  if (Object.keys(configJson).length === 0) {
```

Comment:
do we ever partial config from the `env.NXTP_CONFIG` and a file?

Reply:
we should either update this to error explicitly if both non-empty config and json file are surprised, or combine the configs (probably explicitly error)